src/warnet/control.py

```diff
@@ -198,9 +198,27 @@ def run(scenario_file: str, additional_args: tuple[str]):
     # Create in-memory buffer to store python archive instead of writing to disk
     archive_buffer = io.BytesIO()
 
+    # No need to copy the entire scenarios/ directory into the archive
+    def filter(path):
+        if any(
+            needle in str(path) for needle in [
+                ".pyc",
+                ".csv",
```

Comment:
Do we need to categorically reject csv files? It's a common format, and I can imagine a user thinking they could use it.